packages/components/dialog/src/dialog.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <teleport to="body" :disabled="!appendToBody">
+  <teleport
+    :to="appendTo"
+    :disabled="appendTo !== 'body' ? false : !appendToBody"
+  >
```

Comment:
The logic here seems a bit weird, maybe just keep an `append-to` prop, but removing `appnedToBody` now would also be a breaking change.
Let's keep it like this for now.